tests/rt_ccpp_gsd_unified_ugwp.conf

```diff
@@ -0,0 +1,38 @@
+#################################################################################################################################################################################
```

Comment:
Please merge only the required tests into `rt_ccpp_gsd.conf` and delete `tests/rt_ccpp_gsd_unified_ugwp.conf`.